exampleSite/content/cont/tags.en.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2018-11-29T08:41:44+01:00
+title: Tags
+weight: 40
+tags: ["documentation", "tutorial"]
+---
+
+*Learn theme* support one default taxonomy of gohugo : the *tag* feature.
+
+## Configuration 
+
+Just add tags to any page : 
+
+```markdown
+---
+date: 2018-11-29T08:41:44+01:00
+title: Theme tutorial
+weight: 15
+tags: ["tutorial", "theme"] 
+---
+```
+
+## Behavior
+
+
+The tags are displayed at the top of the page, in the order wich they are enterted.
```

Comment:
```
- wich
+ which
```

```
- enterted
+ entered
```

Not even sure the use of the verb `enter` is correct here. Maybe “in order of insertion” is slightly better?